src/wizard/javascript/browser.md

```diff
@@ -28,16 +28,15 @@ Sentry.init({
     new Integrations.BrowserTracing(),
   ],
 
-  // We recommend adjusting this value in production, or using tracesSampler
-  // for finer control
   tracesSampleRate: 1.0,
 });
 ```
+We recommend adjusting the value of `tracesSampleRate` in production. Learn more about configuring sampling in our [full documentation](https://docs.sentry.io/platforms/javascript/performance/sampling/).
 
 This snippet includes an intentional error, so you can test that everything is working as soon as you set it up:
 
 ```js
 myUndefinedFunction();
 ```
 
-To view and resolve this intentional error, log into sentry.io and open your project. Clicking on the error's title will open a page where you can see detailed information and mark it as resolved.
+Sentry will send an email alerting you to this intentional error; follow the instructions in the email to resolve the error.
```

Comment:
I don't know that we should be recommending that people "mark it as resolved". I mean, the test event, I guess?

I just don't know that we should be describing the workflow right now. The app does a tour once you land on the Issues page. We just want them to send an event.

> Sentry will send an email alerting you to this intentional error; follow the instructions in the email to resolve the error.

Note this wizard is for both new users and existing users creating a new project (and they can disable alerts). So I don't think we should communicate "you will get an email". New users will, existing users may not.

Reply:
Yes, the test event. You're correct that the language is unclear here - thanks for a second set of eyes on this.  

I am a little unclear about the email. In the previous comment, you mentioned that "Note that if they send this error, they will get an email that will bring them into their account, we don't need to give them instructions that they'll forget."

So, perhaps what we mean to say here is:
"If you're new to Sentry, we'll send you an email to access your account where you can complete a tour of the Issues page." And we don't need more for existing users since we can assume they know the flow here. What do you think?